Negative test cases added in api for repository

piyush110786 opened the pull request:
Following test case has been added:
1. New user try to delete repository created by other user.
2. Super user try to delete repository created by other user.

Commit:
Negative test cases added in api for repository

Following test case has been added:

1. New user try to delete repository created by other user.
2. Super user try to delete repository created by other user.

sorenisanerd commented on aasemble/django/apps/api/tests.py:
```diff
@@ -146,6 +146,18 @@ def test_delete_repository_invalid_token(self):
         response = self.client.delete(repo['self'])
         self.assertEquals(response.status_code, 401)
 
+    def test_delete_repository_new_user(self):
```
Maybe rename it to `test_delete_repository_other_user`. The fact that Aaron is a new user isn't the important part. It's important that it's not his repository.

Commit:
Negative test cases added in api for repository

Following test case has been added:

1. New user try to delete repository created by other user.
2. Super user try to delete repository created by other user.

sorenh commented:
Superseded by #68.
